From 04ad293f800039c0221ff010236e589a4ba7079d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Sun, 23 Jun 2024 14:54:42 +0200
Subject: [PATCH] Update test skips for Python 3.12+

Python 3.12 moves testing for `Infinity` and other constants
into `test_json/test_decode.py::test_parse_constant`.
---
 commentjson/tests/test_json/test_decode.py | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/commentjson/tests/test_json/test_decode.py b/commentjson/tests/test_json/test_decode.py
index 6971c44..8f5e904 100644
--- a/commentjson/tests/test_json/test_decode.py
+++ b/commentjson/tests/test_json/test_decode.py
@@ -69,3 +69,8 @@ def test_string_with_utf8_bom(self):
                       'test case is not supported by commentjson.'))
     def test_negative_index(self):
         pass
+
+    @unittest.skipIf(version >= (3, 12),
+                     'Infinity as a value is not supported yet')
+    def test_parse_constant(self):
+        pass
